src/Assets/lib.mo

```diff
@@ -38,7 +38,7 @@ module {
         };
 
         // Determine whether an asset has a given tag.
-        private func _assetHasTag (
+        private func _assetHashTag (
```

Comment:
Checks to see if an asset has a tag. Does not do any hash things :)

Reply:
Oops hehe